commands/global/new.js

```diff
@@ -65,8 +65,9 @@ module.exports.handler = function handler(options) {
       name: projectName,
       version: '0.1.0',
       scripts: {
-        test: 'codemod-cli test',
+        test: 'codemod-cli test --coverage',
```

Comment:
Does running with `--coverage` have an impact on the time to run jest? Since they are internally using istanbul, I would assume it has to add a bit of overhead.

Should we swap this back and add a new?

```js
'test:coverage': 'codemod-cli test --coverage',
```

Then update the `.travis.yml` to use `test:coverage` instead...

Reply:
Yes, makes sense, the coverage part does include a little bit extra time after running the tests.
Updated the PR with the new suggestions.